daemon/config/config_windows.go

```diff
@@ -4,6 +4,12 @@ import (
 	"github.com/docker/docker/api/types"
 )
 
+const (
+	// This is used by the `default-runtime` flag in dockerd as the default value.
+	// On windows we'd prefer to keep this empty so the value is auto-detected based on other options.
+	StockRuntimeName = ""
```

Comment:
The whole `StockRuntimeName` vs `DefaultRuntimeName` is a bit confusing.

Wondering, will setting this to an empty string cause some odd behavior if somehow the option is set to an empty string? https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/opts/runtime.go#L44-L46

Reply:
Ah, never mind; looks like we first check if it's not empty; https://github.com/moby/moby/blob/7b9275c0da707b030e62c96b679a976f31f929d3/opts/runtime.go#L39-L41